README.md

```diff
@@ -24,12 +24,19 @@ run the examples too.
 Building libavif requires [CMake](https://cmake.org/).
 
 No AV1 codecs are enabled by default. Enable them by enabling any of the
-following CMake options:
-
-* `AVIF_CODEC_AOM` - requires CMake, NASM
-* `AVIF_CODEC_DAV1D` - requires Meson, Ninja, NASM
-* `AVIF_CODEC_LIBGAV1` - requires CMake, Ninja
-* `AVIF_CODEC_RAV1E` - requires cargo (Rust), NASM
+following CMake options (e.g. `-DAVIF_CODEC_AOM=ON`):
+
+* `AVIF_CODEC_AOM` for [libaom](https://aomedia.googlesource.com/aom/) (encoder
+  and decoder): local build requires CMake, NASM
```

Comment:
Nit: "local build" may refer to locally building libavif and/or aom. "Locally building aom requires ..." may be more precise.